api/r/cellxgene.census/vignettes/census_axis_query.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Axis Query Example"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{census_axis_query}
```

Comment:
This field should match the title, otherwise pkgdown will complain and generate a warning.